css-canvas-medical/styles.css

```diff
@@ -0,0 +1,247 @@
+
+body {
+  font-family: sans-serif;
+  font-size: 12px;
+  border: 4px solid red;
+}
+
+p,
+li,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+h1 > p {
+  width: 80%;
+  line-height: 2rem;
+  margin-top: 4rem;
+}
+
+h2 {
+  padding-top: 0;
+  margin-top: 0;
+  font-weight: bold;
+}
+
+#big-container {
+  height: auto;
+  width: 1200px;
+  align-items: center;
+  margin-left: 10%;
+  margin-right: 10%;
+  padding-top: 2%;
+}
+
+#container3 > .physician2 {
+  margin: 0;
+  padding: 0;
+}
+
+ul > a {
+  text-decoration: none;
+  color: gray;
+  font-weight: bold;
+}
+
+p > a {
+  text-decoration: none;
+  color: white;
+}
+
+#container,
+#container2 {
+  width: 100%;
+}
+
+.header-items {
+  padding-top: 1%;
+  border-bottom: 1px solid grey;
+  font-weight: lighter;
+}
+
+.header-logo {
+  position: relative;
+  width: 155px;
+  margin-top: 0;
+  margin-left: 20px;
+}
+
+.wrapper {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  margin-left: 45%;
+  right: 10px;
+  bottom: 30px;
+}
+
+.contact {
+  background-color: rgb(35, 35, 126);
+  color: white;
+  border-radius: 5%;
+  width: 110px;
+  height: 2rem;
+}
+
+#container2 {
+  margin-top: 2%;
+  margin-bottom: 4%;
+  width: 100%;
+  text-align: center;
+  position: relative;
+}
+
+.heading-1 {
```

Comment:
Absolute positioning each of these items individually is a lot of work, and can be improved. Instead of moving each element overlaying the hero image individually, create a container for the contents that can be absolute positioned, and simply position that element on top of the image. One element needing to be absolute positioned is far more preferable to many.